docs/modules/ROOT/examples/lite-member-env.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  liteMember:
+    count: 2
```

```suggestion
apiVersion: hazelcast.com/v1alpha1
kind: Hazelcast
metadata:
  name: hazelcast-sample
spec:
  clusterSize: 3
  repository: 'docker.io/hazelcast/hazelcast-enterprise'
  licenseKeySecretName: hazelcast-license-key
  liteMember:
    count: 2
```
I think we should put `licenseKeySecretName` parameter into sample YAMLs otherwise they will not be funtional YAMLs

And also it would be great if you can mention relationship between `clusterSize` and `liteMember.count`,

---

https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/248/commits/eb37f18268899784d0306cafbefbab9c409566b3